python/ray/serve/scripts.py

```diff
@@ -226,6 +227,7 @@ def deploy(config_file_name: str, address: str):
     ),
 )
 @click.argument("config_or_import_path")
+@click.argument("arguments", nargs=-1, required=False)
```

Comment:
nit: add default value? otherwise have to handle it in `_convert_args_to_dict`

Reply:
this already defaults to an empty list